pkg/kv/kvserver/rditer/testdata/TestSelect/replicatedid_span

```diff
@@ -0,0 +1,17 @@
+echo
+----
+Spans():
+- /Local/RangeID/123/{r""-s""}
+- /Local/Range"{a"-c"}
+- /Local/Lock/Intent/Local/Range"{a"-c"}
+- /Local/Lock/Intent"{a"-c"}
+- {a-c}
+StateMachineSpans():
+- /Local/RangeID/123/{r""-s""}
+- /Local/Range"{a"-c"}
+- /Local/Lock/Intent/Local/Range"{a"-c"}
+- /Local/Lock/Intent"{a"-c"}
+- /Local/Lock/Intent/Local/Range"{a"-c"}
+- /Local/Lock/Intent"{a"-c"}
```

Comment:
Why are these duplicated?

Reply:
Because there was a bug - I was appending to the wrong slice in one place in `Select`. Thanks for catching this, I should've found this myself.